app/views/spree/products/_phone_cart.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="phone-visible">
+    <div class="product-options-header d-flex align-items-center justify-content-between">
```

Comment:
Please dynamic this partial as per cart_form. Please discuss this with umer. Also ask umer to guide you to add devise check on this partial. 

This partial should render on mobile devise only